### DISCUSSION TITLE
VAT number field

### DISCUSSION THREAD

**Discussion by mojzesz777** [woocommerce/woocommerce#51177] [Checkout Flow]
How can I add a field with a tax number for the buyer as a company. I have an optional field called "Company" but there is no VAT tax number.

**Comment by alexflorisca** [comment]
You can add an additional field to your checkout [(docs)](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/) or you can use an existing plugin that adds a VAT number field for you ([EU VAT Number for WooCommerce](https://woocommerce.com/products/eu-vat-number/)).

In the future, can I request that questions like this be directed either to the [WC Slack](https://woocommerce.com/community-slack/), where there are lots of active WC developers active in helping the community. Or to the [WooCommerce Forums on Wordpress.org](https://wordpress.org/support/plugin/woocommerce/)

**Comment by nielslange** [comment]
Hello @mojzesz777 👋

As @alexflorisca mentioned, you can use the [additional fields API](https://developer.woocommerce.com/docs/cart-and-checkout-additional-checkout-fields/) for adding fields to the Checkout form. Here's an example code snippet, that should help you getting started:

```php
// Add VAT Number below the Order information section on the Checkout block.
add_action(
    'woocommerce_init',
    function () {
        woocommerce_register_additional_checkout_field(
            array(
                'id'       => 'namespace/vat-number',
                'label'    => 'VAT Number',
                'location' => 'order',
            )
        );
    }
);
```

For sanitisation, validation, etc., please consult the docs that @alexflorisca mentioned before. You can add this snippet to your site using the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin.
